contrib/linux/actions/rsync.yaml

```diff
@@ -9,21 +9,20 @@
             type: 'string'
             description: 'List of files/directories to to be copied'
             required: true
-            position: 1
+            position: 0
```

Comment:
I think the runner ignores all the other values so it does not really matter much.